services/ymax-planner/src/pending-tx-manager.ts

```diff
@@ -79,14 +86,26 @@ const cctpMonitor: PendingTxMonitor<CctpTx, EvmContext> = {
       ctx.evmProviders[caipId] ||
       Fail`${logPrefix} No EVM provider for chain: ${caipId}`;
 
-    const transferStatus = await watchCctpTransfer({
-      usdcAddress,
-      watchAddress: accountAddress as `0x${string}`,
-      expectedAmount: amount,
-      provider,
-      log: (msg, ...args) => log(`${logPrefix} ${msg}`, ...args),
-      timeoutMs,
-    });
+    let transferStatus = false;
+    if (opts.mode === 'live') {
+      transferStatus = await watchCctpTransfer({
+        usdcAddress,
+        toAddress: accountAddress as `0x${string}`,
+        expectedAmount: amount,
+        provider,
+        log: (msg, ...args) => log(`${logPrefix} ${msg}`, ...args),
```

Comment:
string concatenation seems avoidable:

```suggestion
        log: (msg, ...args) => log(logPrefix, msg, ...args),
```

also: consider using `makeTracer(...)` and `tracer.sub(...)`